modules/insights-operator-new-pull-secret-disabled.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * support/insights-operator/disabling-insights-operator.adoc
+
+[id="insights-operator-new-pull-secret_{context}"]
+= Modifying the global cluster pull secret to disable Telemetry and the Insights Operator
+
+You can modify your existing global cluster pull secret to disable Telemetry and the Insights Operator.
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. Download the global cluster pull secret to your local file system.
++
+----
+$ oc extract secret/pull-secret -n openshift-config --to=.
+----
+
+. In a text editor, edit the `.dockerconfigjson` file that was downloaded.
+
+. Remove the `cloud.openshift.com` JSON entry, for example:
++
+[source,json]
+----
+"cloud.openshift.com":{"auth":"<hash>","email":"<email_address>"}
```

Comment:
@smarterclayton I added this example since you last looked at it, if you can verify this is okay to show like this.

Preview: https://osdocs-660-insights--ocpdocs.netlify.com/openshift-enterprise/latest/support/insights-operator/disabling-insights-operator.html#insights-operator-new-pull-secret_insights-operator-disabling